### ISSUE TITLE
Language translate on v4.0.2 

### ISSUE THREAD

**Issue by krisvandermerwe** [glitch-soc/mastodon#1968]
### Pitch

I have heard that Mastadon v4.0.2 has a language translation button/link for toots?

Is there a current plans to upgrade  "social.dev-wiki.de"  to  v4.0.2 ?

### Motivation

This a core feature that is available on all major social platforms
Strictly necessary for international/federated platforms

**Comment by ClearlyClaire** [comment]
> I have heard that Mastadon v4.0.2 has a language translation button/link for toots?

Yes. This requires the server admins to configure a translation backend (likely to be an external service, possibly a paid one), though. Also, that interface is currently not available in the glitch-soc flavor.

> Is there a current plans to upgrade "social.dev-wiki.de" to v4.0.2 ?

That is not something the glitch-soc project can answer, please reach out to the `social.dev-wiki.de` admins.

**Comment by hackdefendr** [comment]
> Also, that interface is currently not available in the glitch-soc flavor.

So setting up Libretranslate locally on my server won't work in the glitch-soc flavor of Mastodon? Any chance that will eventually make it in this flavor?

Regards,
Jeff

**Comment by ClearlyClaire** [comment]
Right, it would be available in the vanilla front-end flavor and in supporting apps, but not in the glitch-soc flavor.

This is planned, see #1865
